src/libraries/Common/src/System/Net/SocketProtocolSupportPal.Windows.cs

```diff
@@ -10,13 +10,14 @@ internal static partial class SocketProtocolSupportPal
     {
         private static bool IsSupported(AddressFamily af)
         {
+            const int StreamSocketType = 1;
             Interop.Winsock.EnsureInitialized();
 
             IntPtr INVALID_SOCKET = (IntPtr)(-1);
             IntPtr socket = INVALID_SOCKET;
             try
             {
-                socket = Interop.Winsock.WSASocketW(af, DgramSocketType, 0, IntPtr.Zero, 0, (int)Interop.Winsock.SocketConstructorFlags.WSA_FLAG_NO_HANDLE_INHERIT);
```

Comment:
This broke `Socket.OSSupportsUnixDomainSockets` as Windows only support Stream for UDS.